### PR TITLE
Objects box tests and entities added

### PR DESCRIPTION
**Pull request by EugeneDar**
Closes: #25

---

## PR-Codex overview
This PR introduces a new `Entity` class and modifies the `DummyTest` class. It also adds new test classes and modifies the `ObjectsBox` class. 

### Detailed summary
- `Entity` class added to `org.objectionary.entities` package
- `DummyTest` class modified to extend `Entity` class
- `Empty` and `Data` classes added to `org.objectionary.entities` package
- New test classes added: `ObjectsBoxTest`, `EmptyTest`, and `DataTest`
- `ObjectsBox` class modified to include new methods and fields for adding and retrieving objects from the box.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by EugeneDar** [comment]
@Graur сhecks have not started for an hour, even making another commit did not help. Maybe you know how to fix it?

**Comment by EugeneDar** [comment]
@Graur please, have a look

File: src/main/java/org/objectionary/ObjectsBox.java [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    public ObjectsBox() {
+        this.box = new HashMap<>();
+    }
+
+    /**
+     * Puts an object into the box.
+     * @param name The name of the object.
+     * @param bindings The bindings of the object.
+     */
+    public void putObject(final String name, final Map<String, Entity> bindings) {
```
Comment (Graur) [inline src/main/java/org/objectionary/ObjectsBox.java:53]: @EugeneDar Can we simplify this object name? According to: https://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html

File: src/main/java/org/objectionary/ObjectsBox.java [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        this.box.put(name, bindings);
+    }
+
+    /**
+     * Gets an object by name.
+     * @param name The name of the object.
+     * @return The object.
+     */
+    public Map<String, Entity> getObject(final String name) {
```
Comment (Graur) [inline src/main/java/org/objectionary/ObjectsBox.java:62]: @EugeneDar Same here: https://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html

**Review by Graur — CHANGES_REQUESTED** [review]

**Comment by Graur** [comment]
> @Graur сhecks have not started for an hour, even making another commit did not help. Maybe you know how to fix it?

@EugeneDar Can you check again? Looks like it was temporary problem

File: src/main/java/org/objectionary/ObjectsBox.java (hunk 1 above)
Comment (EugeneDar) [reply src/main/java/org/objectionary/ObjectsBox.java]: @Graur sure, I will fix it

File: src/main/java/org/objectionary/ObjectsBox.java (hunk 2 above)
Comment (EugeneDar) [reply src/main/java/org/objectionary/ObjectsBox.java]: @Graur sure, I will fix it

**Comment by EugeneDar** [comment]
@Graur please, have a look again

**Review by Graur — APPROVED** [review]
@EugeneDar Good one
@yegor256 Please, have a look

**Comment by EugeneDar** [comment]
@yegor256 please, have a look

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**